install/charts/templates/delete-clean-up.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+# This file is licensed under the Apache Software License, v. 2 except as
+# noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.metadata.name }}-delete
+  labels:
+    app: {{ .Values.metadata.labelApp }}
+  namespace: {{ .Values.metadata.namespace }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+spec:
+  template:
+    metadata:
+      name: {{ .Values.metadata.name }}-pre-delete
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount }}
+      restartPolicy: OnFailure
+      containers:
+        - name: {{ .Values.metadata.name }}-pre-delete-container
+          image: "lachlanevenson/k8s-kubectl"
+          command: ['sh', '-c', 'kubectl delete mutatingwebhookconfigurations karydia-webhook && kubectl delete validatingwebhookconfigurations karydia-webhook && kubectl delete customresourcedefinitions karydianetworkpolicies.karydia.gardener.cloud && kubectl delete customresourcedefinitions karydiaconfigs.karydia.gardener.cloud']
```

Comment:
This block is very hard to read. What do you think of structuring it more like in https://github.com/karydia/karydia/blob/4087d0992fae982035889730a76829facd3cb496/install/charts/templates/workaround-annotate-kube-system-namespace.yaml#L42-L50

And if I remember correctly it is possible to replace the space between the component's type and name with `/` and separate multiple deletable objects with space within one `kubectl delete` command. Something like the following (untested):
```
kubectl delete mutatingwebhookconfigurations/karydia-webhook validatingwebhookconfigurations/karydia-webhook customresourcedefinitions/karydianetworkpolicies.karydia.gardener.cloud customresourcedefinitions/karydiaconfigs.karydia.gardener.cloud
```

Reply:
Updated, to look more like the suggested format.